Runtime/EcsactDefaults.cs

```diff
@@ -0,0 +1,59 @@
+using System.Runtime.CompilerServices;
+using System;
+
+#nullable enable
+
+[assembly: InternalsVisibleTo("EcsactRuntimeDefaults")]
+
+namespace Ecsact {
+	static public class Defaults {
+
+		internal static EcsactRuntime? _Runtime;
+		public static EcsactRuntime Runtime => _Runtime
+			?? throw new Exception(
+				"Runtime is null, if you want to access it as early as possible" +
+				" use Ecsact.Defaults.WhenReady"
+			);
+		internal static Ecsact.Registry? _Registry;
+		public static Ecsact.Registry Registry => _Registry 
+			?? throw new Exception(
+				"Registry is null, if you want to access it as early as possible" +
+				" use Ecsact.Defaults.WhenReady"
+			);
+		public static Ecsact.UnitySync.EntityGameObjectPool? Pool;
+		public static EcsactRunner? Runner;
+
+		private static event global::System.Action? onReady;
+
+		public static void WhenReady
+			( global::System.Action callback
+			)
+		{
+			if(Runtime != null) {
+				callback();
+			} else {
+				onReady += callback;
+			}
+		}
+
+		internal static void NotifyReady() {
+			if(Runtime != null) {
+				onReady!.Invoke();
```

Comment:
with C# events if there are no listeners the event will be null.
```suggestion
				onReady?.Invoke();
```